File: SmallCapital.cpp
#include <iostream>
using namespace std;
// method
void small_cap(int fin){  //void is return type.
    char result;
    
      if(fin>='a' && fin<='z'){
        
        result = fin- 'a' + 'A';
    }
    
    cout<<result;
}

// main function 
int main() {
  
    char fin ;

    cout<<"enter any small letter alphabet ";
    cin>>fin;
    small_cap(fin);
   

File: SmallCapital.java
import java.util.Scanner;

public class SmallCapital {
    
    public static void small_cap(char fin) {
        if (fin >= 'a' && fin <= 'z') {
            // Convert the lowercase letter to uppercase
            fin = (char) (fin - 'a' + 'A');
        }
        
        System.out.println(fin);
    }
    
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        System.out.println("Enter any lowercase letter: ");
        char fin = scanner.next().charAt(0);
        
        small_cap(fin);
        
        scanner.close();
    }
}
